airbyte-ci/connectors/metadata_service/lib/metadata_service/models/generated/GeneratedFields.py

```diff
@@ -0,0 +1,35 @@
+# generated by datamodel-codegen:
+#   filename:  GeneratedFields.yaml
+
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, Extra, Field
+
+
+class GitInfo(BaseModel):
+    class Config:
+        extra = Extra.forbid
+
+    commit_sha: Optional[str] = Field(
+        None,
+        description="The git commit sha of the last commit that modified this file. DO NOT DEFINE THIS FIELD IN YOUR SPEC. It will be overwritten by the CI.",
```

Comment:
What do you mean by SPEC in this context? Do you mean the `metadata.yaml` file? 
Sounds confusing to me as `spec` also relates to the connector spec command.

Reply:
Oh good catch. Mistype Ill change to "metadata file"